lua/gitlab/actions/discussions/signs.lua

```diff
@@ -136,7 +263,10 @@ M.refresh_signs = function(discussions)
         discussion_line = first_note.position.old_line
         start_line = start_old_line
         end_line = end_old_line
+      else
+        return {}, {}, string.format("Unsupported line range type found for discussion %d", discussion.id)
```

Comment:
I'm adding this as a safeguard. Probably should not return but having an explicit error pattern is nice.